examples/interpreter-in-browser/www/chain/Elixir.Chain.erl.old

```diff
@@ -0,0 +1,61 @@
+-file("lib/chain.ex", 1).
```

Comment:
Why are we keeping the `.old` version in source control?

Reply:
That's my mistake. As I mentioned, I think I would prefer not keeping these files in the repo at all.